app/models/commits_service.rb

```diff
@@ -0,0 +1,8 @@
+class CommitsService
+
+  def self.find_commits(username, repository)
+    response = Faraday.get("https://api.github.com/repos/#{username}/#{repository}/commits?client_id=#{ENV['client_id']}&client_secret=#{ENV['client_secret']}")
```

Comment:
Yup! I tried that and it works on LocalHost, I just couldnt get it to pass my tests, is all! @case-eee